tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Models/ResultTests.cs

```diff
@@ -54,5 +95,44 @@ public void TwoDifferenceResults_AreNotEqual()
 
             actual.Should().BeFalse();
         }
+
+        [Test]
+        public void TwoDifferentResultsT_AreNotEqual()
+        {
+            var success = Result.Success("TestA");
+            var failure = Result.Success("TestB");
+
+            var actual = success.Equals(failure);
+
+            actual.Should().BeFalse();
+        }
+
+        [Test]
+        public void ToResult_ConvertSuccessResultT_ReturnsSuccessResult()
+        {
+            var sut = Result.Success("Test");
+            
+            var actual = sut.ToResult();
+
+            actual.Should().Be(Result.Success());
+        }
+
+        [Test]
+        public void ToResult_ConvertFailureResultT_ReturnsFailureResult()
+        {
+            List<ErrorMessage> expectedErrors = new List<ErrorMessage> { new ErrorMessage("TestErrorId") };
+            var sut = Result.Failure<string>(expectedErrors);
```

Comment:
`sut` as above.

Reply:
Same reply as above. Will look at this after the coding standards meeting but you are right to point it out 👍